webgame-game/src/gridworld.rs

```diff
@@ -50,6 +53,19 @@ impl LevelLayout {
     }
 }
 
```

Comment:
All agents now have an `Agent` component that contains common state across all agents (e.g. which direction they're looking at).